src/main/java/cz/startnet/utils/pgdiff/PgDiff.java

```diff
@@ -187,32 +191,36 @@ private static void diffDatabaseSchemas(final PrintWriter writer,
      * @param writer      writer the output should be written to
      * @param oldDatabase original database schema
      * @param newDatabase new database schema
+     * @param arguments object containing arguments settings
      */
     private static void dropOldSchemas(final PrintWriter writer,
-            final PgDatabase oldDatabase, final PgDatabase newDatabase) {
+            final PgDatabase oldDatabase, final PgDatabase newDatabase,
+            final PgDiffArguments arguments) {
         for (final PgSchema oldSchema : oldDatabase.getSchemas()) {
             if (newDatabase.getSchema(oldSchema.getName()) == null) {
                 writer.println();
-                writer.println("DROP SCHEMA "
+                writer.println("DROP SCHEMA " +PgDiffUtils.getDropIfExists(arguments.isUseIfExists())
```

Comment:
we're passing this argument around too much -- could we have this as a config param for PgDiffUtils so that we don't need to pass it every time?